grim_test.go

```diff
@@ -212,6 +212,35 @@ func TestHookGetsLogged(t *testing.T) {
 
 }
 
+func TestShouldSkip(t *testing.T) {
```

Comment:
wouldnt this not be possible to occur? In the extractHookEvent method, it makes the EventName either a pull_request or a push
if parsed.Action != "" {
		hook.EventName = "pull_request"
		. . .
	} else {
		hook.EventName = "push"
		. . .
	}
(from https://github.com/MediaMath/grim/blob/master/github_hook.go#L110 )